### PR TITLE
Update renovate.json

### PR DESCRIPTION
**Pull request by at-the-vr**
updates the Renovate Bot configuration to improve the scheduling of dependency updates (anytime for `astro` & `astrojs` related dependencies, Mondays for rest other dependencies)

### CODE REVIEW

**Review by florian-lefebvre — APPROVED** [review]
Let's try!

**Comment by alexanderniebuhr** [comment]
Happy to try it, did you find some docs which states that the order matters? 🤔

File: .github/renovate.json [hunk 1]
```diff
@@ -6,19 +6,19 @@
...
       "schedule": ["at any time"]
     },
     {
       "matchPackagePatterns": ["^@astrojs"],
       "schedule": ["at any time"]
+    },
+    {
+      "groupName": "all dependencies",
+      "groupSlug": "all",
+      "matchPackagePatterns": ["*"],
```
Comment (alexanderniebuhr) [inline .github/renovate.json:20]:
```suggestion
      "matchPackagePatterns": ["*"],
			"excludePackagePatterns": ["^@astrojs", "^astro"],
```

File: .github/renovate.json [hunk 2]
```diff
@@ -6,19 +6,19 @@
...
-      "schedule": ["before 4am on Monday"]
-    },
     {
       "matchPackagePatterns": ["^astro"],
       "schedule": ["at any time"]
     },
     {
       "matchPackagePatterns": ["^@astrojs"],
       "schedule": ["at any time"]
+    },
```
Comment (alexanderniebuhr) [inline .github/renovate.json:16]:
```suggestion
    {
      "matchPackagePatterns": ["^@astrojs", "^astro"],
      "schedule": ["at any time"]
    },
```

**Review by alexanderniebuhr — COMMENTED** [review]
Maybe this would work even better?

**Comment by alexanderniebuhr** [comment]
In addition would it be possible to re-open this PR on a branch with the name `renovate/reconfigure`, so we get config file validation? (https://docs.renovatebot.com/config-validation/#validation-of-renovate-config-change-prs)

**Comment by at-the-vr** [comment]
on it, thanks for the huge saves 😅

**Pull request closed without merging**